lib/cisco_node_utils/cmd_ref/span_session.yaml

```diff
@@ -0,0 +1,64 @@
+# SPAN session
+---
+_template:
+  nexus:
+    get_command: "show running monitor all"
+
+all_sessions:
+  multiple:
+  get_context: ~
+  get_value: '/^monitor session (\d+)$/'
+
+create:
+  set_value: 'monitor session %s'
+
+description:
+  kind: string
+  get_context: ["monitor session <id>"]
+  get_value: '/^description (.*)/'
+  set_context: ["monitor session <id>"]
+  set_value: '<state> description <desc>'
+  default_value: ''
+
+destination:
+  kind: string
+  get_context: ["monitor session <id>"]
+  get_value: '/^destination interface <intf_name>$/'
+  set_context: ["monitor session <id>"]
+  set_value: '<state> destination interface <intf_name>'
+
+destroy:
+  set_value: 'no monitor session %s'
+
+session_id:
+  kind: int
+  default_value: false
+  get_value: '/^monitor session (\d+)$/'
```

Comment:
You need to remove the `$` in this case as well so it matches lines with and without the type.